README.md

```diff
@@ -1,4 +1,4 @@
-# netuitive.packages.collectd 1.1.1
+# CloudWisdom.packages.collectd 1.1.1
```

Comment:
Keep this as netuitive, it's part of a fully qualified name.

Reply:
changed back to netuitive